paddle/go/pserver/service.go

```diff
@@ -135,7 +138,10 @@ func (s *Service) SendGrads(grads []Gradient, dummy *int) error {
 	errCh := make(chan error, count)
 	for _, g := range grads {
 		go func(p Parameter, g Gradient) {
-			err := s.opt.UpdateParameter(p, g)
+			opt, err := s.optimizerMap[p.Name]
+			if err != nil {
+				err := opt.UpdateParameter(p, g)
```

Comment:
Sadly I keep do my job in baidu's dev machine, last two days "jumbo(a package management system)" broken in whole baidu. It do not pass the go compile side. 
Thanks for your lint package!  I use spacemacs, what a good news to me!